src/test/java/com/uid2/shared/store/SiteTest.java

```diff
@@ -1,23 +1,32 @@
 package com.uid2.shared.store;
 
+import com.uid2.shared.model.ClientType;
 import com.uid2.shared.model.Site;
 import org.junit.jupiter.api.Test;
 
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.HashSet;
 
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
 
 public class SiteTest {
     @Test
     public void testEquals() {
-        Site a = new Site(1, "1 name", "1 desc", true, new HashSet<>(), new HashSet<>(), true, 0L);
-        Site b = new Site(1, "1 name", "1 desc", true, new HashSet<>(), new HashSet<>(), true, 0L);
-        Site c = new Site(1, "1 name", "1 desc", true, new HashSet<>(), new HashSet<>(), null, 0L);
+        Site a = new Site(1, "1 name", "1 desc", true, new HashSet<>(), new HashSet<>(), new HashSet<>(), true, 0L);
+        Site b = new Site(1, "1 name", "1 desc", true, new HashSet<>(), new HashSet<>(), new HashSet<>(), true, 0L);
+        Site c = new Site(1, "1 name", "1 desc", true, new HashSet<>(), new HashSet<>(), new HashSet<>(), null, 0L);
```

Comment:
not directly related a result of your work but would be great if you could beef up the equal tests here so we should have `equal` test cases for checking non empty client types/domain name/app name sets too